dist/sails.io.js

```diff
@@ -1492,10 +1535,18 @@ return new(b[["Active"].concat("Object").join("X")])("Microsoft.XMLHTTP")}catch(
       // The environment we're running in.
       // (logs are not displayed when this is set to 'production')
       //
-      // Defaults to development unless this script was fetched from a URL
-      // that ends in `*.min.js` or '#production' (may also be manually overridden.)
-      //
-      environment: urlThisScriptWasFetchedFrom.match(/(\#production|\.min\.js)/g) ? 'production' : 'development',
+      // Defaults to "development" unless this script was fetched from a URL
+      // that ends in `*.min.js` or '#production', or if the conventional
+      // `SAILS_LOCALS` global is set with an `_environment` of "production"
+      // or "staging".  (This setting may also be manually overridden.)
+      environment: (
+        urlThisScriptWasFetchedFrom.match(/(\#production|\.min\.js)/g) ||
```

Comment:
@sgress454 nice nice nice-- was just beginning to think about the same thing when I noticed this

This makes it fully automated for new v1.x Sails apps, no matter how the pipeline of choice ends up squirting out those asset bundles

Reply:
ha i think you wrote this code but thanks -- this changeset is probably just from dist being recreated.